documentation/src/main/asciidoc/topics/ref_performance_tuning.adoc

```diff
@@ -79,6 +79,92 @@ The recommended values are:
 * send window size - 640 KB
 * receive window size - 25 MB
 
+== Thread pool settings
+
+A thread pool manages threads by assigning them concurrent tasks.
+Additionally, a thread pool can improve system performance by using any idle thread instead of creating threads.
+
+Thread pools limit the creation and deletion of threads to active worker threads, so that the amount of threads does not typically exceed the amount of tasks.
+This helps reduce the memory footprint of your {brandname} server.
+
+You can configure your `threading` subsystem, so that you can better manage thread pools and their resources. 
+
+The following table outlines common `threading` subsystem components that you can configure:
+
+// Resource: https://docs.jboss.org/infinispan/13.0/configdocs/infinispan-config-13.0.html
+
+[cols="1,1,3"]
+|===
+| Component  | Attribute type |Description
+
+|`blocking-bounded-queue-thread-pool`|string and integer | A thread pool executor that can run blocking operations. 
+Executor checks that the number of running threads does not exceed the core size; otherwise, the executor creates new threads. 
+Idle threads are placed in a queue.
```

Comment:
This line doesn't make much sense. 

I would just be a bit more generic and say something to the extent that the blocking thread pool will do a best effort to keep as minimal amount of threads around, reusing idle ones as much as possible. This differs from the JRE ThreadPoolExecutor that it spawns new threads for each request up to the core size.

Reply:
Updated to:

> This behavior differs from the JRE's `ThreadPoolExecutor` that creates new threads for each generated request until core size limitations apply.

WDYT?